src/meta/src/rpc/ddl_controller.rs

```diff
@@ -843,16 +1037,22 @@ impl DdlController {
         let job_id = stream_job.id();
         tracing::debug!(id = job_id, "creating stream job");
 
-        let result = self
-            .stream_manager
-            .create_streaming_job(table_fragments, ctx)
-            .await;
+        let result = try {
+            // Add table fragments to meta store with state: `State::Initial`.
+            mgr.fragment_manager
```

Comment:
Forward storing table fragments to make `create_streaming_job` in stream manager compatible with two versions of metadata manager.